### PR TITLE
PE-D Bug Fixes

### LINKED ISSUE #76: [PE-D][Tester E] Wrong user guide shown in `help` window
## Description

The user guide shown in the `help` window does not point to the right user guide.

Expected: https://ay2223s1-cs2103t-t11-1.github.io/tp/UserGuide.html

Got: https://se-education.org/addressbook-level3/UserGuide.html

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: shenyih0ng/ped#1

### LINKED ISSUE #80: [PE-D][Tester E] Name of `.jar` file does not match user guide
## Description

Release is named as `addressbook.jar` instead of the `contactmation.jar` mentioned in the user guide (under "Quick Start" section).

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: shenyih0ng/ped#2

### LINKED ISSUE #81: [PE-D][Tester A] Prefix are case sensitive
Prefix are case sensitive, changing from t to T causes the programme to not work

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: PokezardVGC/ped#2

### PR DESCRIPTION
**Pull request by jasonchristopher21**
Fix bugs found in PE-D:
- Bug in user guide link
- Naming of `.jar` file
- Bug in prefix parsing

Fixes #76, Fixes #80, Fixes #81

### CODE REVIEW

**Review by mohamedsaf1 — COMMENTED** [review]
LGTM. Adding equals methods to ensure no repetition in output for same commands.

File: src/main/java/seedu/address/model/attribute/AttributeList.java [hunk 1]
```diff
@@ -245,8 +247,4 @@ public String toString() {
         return builder.toString();
     }
 
-    private String formatProperName(String name) {
-        return name.substring(0, 1).toUpperCase() + name.substring(1).toLowerCase();
-    }
-
```
Comment (Eclipse-Dominator) [inline src/main/java/seedu/address/model/attribute/AttributeList.java:251]: Good idea to consolidate all the parse related methods

File: src/main/java/seedu/address/model/attribute/Email.java [hunk 2]
```diff
@@ -58,4 +58,17 @@ public static boolean isValidEmail(String test) {
...
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o == this) {
+            return true; // short circuit if same object
+        } else if (o instanceof Email) {
+            Email email = (Email) o;
+            String currEmail = value.replaceAll("\\s+", "");
+            String otherEmail = email.value.replaceAll("\\s+", "");
```
Comment (Eclipse-Dominator) [inline src/main/java/seedu/address/model/attribute/Email.java:69]: email shouldn't have spaces if declared properly, so I think this part is not necessary!

File: src/main/java/seedu/address/model/attribute/Name.java [hunk 3]
```diff
@@ -45,4 +45,17 @@ public static boolean isValidName(String test) {
...
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (o == this) {
+            return true; // short circuit if same object
+        } else if (o instanceof Name) {
+            Name n = (Name) o;
+            String currName = fullName.replaceAll("\\s+", "");
+            String otherName = n.fullName.replaceAll("\\s+", "");
```
Comment (Eclipse-Dominator) [inline src/main/java/seedu/address/model/attribute/Name.java:56]: Perhaps it's better to do this conversion in the constructor?
i.e. change `super(TYPE, name)` to `super(TYPE, name.replaceAll("\\s+", " "))`;
This way we dont have to do this additional check?

Comment (Eclipse-Dominator) [reply src/main/java/seedu/address/model/attribute/Name.java:56]: There is also a problem with this is that 
"Joh Natalie" will match "John Atalie"

**Review by Eclipse-Dominator — CHANGES_REQUESTED** [review]
Take note of some errors in the Name.java equals() implementation!

**Review by Eclipse-Dominator — APPROVED** [review]

**Pull request merged**